### DISCUSSION TITLE
Filter if CF has value

### DISCUSSION THREAD

**Discussion by lschnetl** [netbox-community/netbox#15672] [Q&A]
Hi everyone

Is there a way I can filter devices if a custom field has any value. Basically I want to find empty or skipped custom fields that we depend on

Currently there is has_primary_ip and has_oob, so I want to do something similar

**Comment by candlerb** [comment]
Are you talking about the REST API, or the GUI?

For the REST API there is some info in #11538 which might help. (Although that issue is specifically a problem with "selection" custom fields, it gives some filtering examples which may work with regular custom fields). For the GUI it may work if you assemble the correct filter manually in the URL bar.

**Reply by lschnetl** [reply]
From the GUI mainly.

I will play around with that. I just wondered if there was something that I might have overlooked.

My primary use case are for some GUID values that get assigned to hosts from AMT. So running a filter might make it slightly more difficult
